app/controllers/application_controller.rb

```diff
@@ -2,4 +2,21 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+protected
+
+def configure_permitted_parameters
+  devise_parameter_sanitizer.for(:sign_up) <<
+    [:username]
+
+  devise_parameter_sanitizer.for(:account_update) do |u|
+    u.permit(
+      :password,
+      :password_confirmation,
+      :current_password
+    )
+    end
```

Comment:
`end` at 20, 4 is not aligned with `devise_parameter_sanitizer.for(:account_update) do |u|` at 14, 2